src/main/java/edu/arizona/kra/proposaldevelopment/lookup/PropDevRouteStopValueFinder.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2005-2015 The Kuali Foundation
+ * 
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.osedu.org/licenses/ECL-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.arizona.kra.proposaldevelopment.lookup;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.kuali.rice.core.api.util.ConcreteKeyValue;
+import org.kuali.rice.core.api.util.KeyValue;
+import org.kuali.rice.krad.uif.control.UifKeyValuesFinderBase;
+import static edu.arizona.kra.proposaldevelopment.PropDevRoutingStateConstants.*;
+
+@SuppressWarnings("serial")
```

Comment:
This class is serializable by way of its ancestor KeyValuesBase, why not generate a serialVersionUID instead of suppressing it?